structuredheadings/plugin.js

```diff
@@ -94,11 +100,11 @@
           if (element.is(allowedElements)) {
             switch (editor.config.enterMode) {
             case CKEDITOR.ENTER_DIV:
-                  //eslint-disable-next-line new-cap
+              //eslint-disable-next-line new-cap
               style = new CKEDITOR.style({ element: "div" });
               break;
             default:
-                  //eslint-disable-next-line new-cap
+              //eslint-disable-next-line new-cap
```

Comment:
I thought I found a new way of applying the style so I don't need to do this every time, but it didn't work. I'll keep looking as I go.